Packages/Sites/Sfi.Kateheo/Configuration/NodeTypes.yaml

```diff
@@ -82,6 +82,60 @@ blah: &alohaRich
       'Sfi.Listable:ListableMixin': true
       'Sfi.Listable:ListableCategoryMixin': true
 
+'Sfi.Kateheo:Author':
+  superTypes:
+    - 'TYPO3.Neos:Document'
+  ui:
+    label: 'Автор'
+  childNodes:
+    main:
+      type: 'TYPO3.Neos:ContentCollection'
+  properties:
+    title:
+      ui:
+        inlineEditable: true
+        aloha:
+          <<: *alohaPlain
+          placeholder: 'Имя автора'
+    originalIdentifier:
+      type: string
+
+'Sfi.Kateheo:TagsPage':
+  superTypes:
+    - 'TYPO3.Neos:Document'
+    - 'Sfi.Listable:ListableCategoryMixin'
+  ui:
+    label: 'Страница тэгов'
+  childNodes:
+    main:
+      type: 'TYPO3.Neos:ContentCollection'
+  properties:
+    originalIdentifier:
+      type: string
+  constraints:
+    nodeTypes:
+      '*': false
+      'Sfi.Listable:ListableMixin': true
+      'Sfi.Listable:ListableCategoryMixin': true
+
+'Sfi.Kateheo:Tags':
+  superTypes:
+    - 'TYPO3.Neos:Document'
+  ui:
+    label: 'Тэг'
+  childNodes:
+    main:
+      type: 'TYPO3.Neos:ContentCollection'
+  properties:
+    title:
+      ui:
+        inlineEditable: true
+        aloha:
+          <<: *alohaPlain
+          placeholder: 'Тэг'
+    originalIdentifier:
```

Comment:
https://github.com/psmb/KateheoDistr/blob/master/Packages/Sites/Sfi.Kateheo/Classes/Sfi/Kateheo/Command/MigrationCommandController.php#L112
It's needed only in migration script